internal/cli/atlas/events/orgs_list.go

```diff
@@ -25,7 +26,8 @@ import (
 	store "github.com/mongodb/mongodb-atlas-cli/internal/store/atlas"
 	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
 	"github.com/spf13/cobra"
-	atlas "go.mongodb.org/atlas/mongodbatlas"
+	"go.mongodb.org/atlas/mongodbatlasv2"
+	"k8s.io/utils/pointer"
```

Comment:
👀

Reply:
Forgot to disable Github copilot. It does keep introducing Kubernetes dependencies :(.